From 9daeddc31155ed521c2074c76f019b635c062933 Mon Sep 17 00:00:00 2001
From: Richard Sun <richard.sun@mediatek.com>
Date: Fri, 19 Apr 2019 13:09:20 +0800
Subject: [PATCH] fix compiling mtd_probe.c fail and including mtd_probe.h

In file included from extras/mtd_probe/mtd_probe.c:19:0:
extras/mtd_probe/mtd_probe.h:24:2: error: unknown type name 'uint32_t'
  uint32_t reserved;
  ^
extras/mtd_probe/mtd_probe.h:25:2: error: unknown type name 'uint8_t'
  uint8_t data_status;
  ^
extras/mtd_probe/mtd_probe.h:26:2: error: unknown type name 'uint8_t'
  uint8_t block_status;
  ^
extras/mtd_probe/mtd_probe.h:27:2: error: unknown type name 'uint8_t'
  uint8_t lba_copy1[2];
  ^
extras/mtd_probe/mtd_probe.h:28:2: error: unknown type name 'uint8_t'
  uint8_t ecc2[3];
  ^
extras/mtd_probe/mtd_probe.h:29:2: error: unknown type name 'uint8_t'
  uint8_t lba_copy2[2];
  ^
extras/mtd_probe/mtd_probe.h:30:2: error: unknown type name 'uint8_t'
  uint8_t ecc1[3];
  ^
---
 src/udev/mtd_probe/mtd_probe.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/udev/mtd_probe/mtd_probe.h b/src/udev/mtd_probe/mtd_probe.h
index 2a37ede57..5102b1b5e 100644
--- a/src/udev/mtd_probe/mtd_probe.h
+++ b/src/udev/mtd_probe/mtd_probe.h
@@ -18,6 +18,8 @@
  */
 
 #include <mtd/mtd-user.h>
+#include <stdint.h>
+
 
 /* Full oob structure as written on the flash */
 struct sm_oob {
-- 
2.18.0

